src/main/java/build/buildfarm/instance/stub/Chunker.java

```diff
@@ -171,7 +167,7 @@ public Chunk next() throws IOException {
 
     if (digest.getSizeBytes() == 0) {
       data = null;
-      return emptyChunk;
+      return new Chunk(Digest.getDefaultInstance(), ByteString.EMPTY, 0);
```

Comment:
It is a bit of a hack, IIUC. Basically, this works because your usage of Chunker now ensures that Chunkers with empty data never get uploaded, so it doesn't matter what the digest hash is, as long as size_bytes is 0. But I don't have a better idea :-)

Reply:
Yeah, I didn't want to be checking for valid permutations of empty hashes everywhere, and forcing everything to know about that instance-configured method